lib/classes/PluginManager.js

```diff
@@ -19,19 +24,19 @@ const requireServicePlugin = (servicePath, pluginPath, localPluginsPath) => {
     try {
       return require(absoluteLocalPluginPath);
     } catch (error) {
-      if (!isModuleNotFoundError(error, absoluteLocalPluginPath)) throw error;
+      if (error.code !== 'MODULE_NOT_FOUND') throw error;
```

Comment:
Fair enough - in this case I think the right fix is to first make a `require.resolve(absoluteLocalPluginPath)` (which will only throw on `absoluteLocalPluginPath`, since it won't evaluate its code just yet), catch exceptions and check for `code` as I did, then if it works make the actual `require`.

Reply:
That's a very good idea, It'll definitely make this error handling more solid, we're open for such improvement.